src/Tx_main.cpp

```diff
@@ -23,6 +23,8 @@ uint8_t broadcastAddress[6] = {0, 0, 0, 0, 0, 0};
 #endif
 uint8_t bindingAddress[6] = {0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF};
 
+const uint8_t version[] = {LATEST_VERSION};
```

Comment:
Where does `LATEST_VERSION` get defined? Build scripts?

Reply:
Yeah, comes from the build scripts